actioncable/test/test_helper.rb

```diff
@@ -66,8 +66,11 @@ class ActionCable::TestCase < ActiveSupport::TestCase
   end
 
   def wait_for_executor(executor)
-    until executor.completed_task_count == executor.scheduled_task_count
+    # do not wait forever
+    timeout = 2
+    until timeout > 0 && executor.completed_task_count == executor.scheduled_task_count
```

Comment:
This seems fair :+1:. But let's raise if we reach the timeout, so we're not relying on the surrounding test noticing the problem.